lib/index.js

```diff
@@ -30,10 +31,17 @@ Outbound.prototype.initialize = function() {
   window.outbound.methods = [
     'identify',
     'track',
+    'alias',
     'registerApnsToken',
     'registerGcmToken',
     'disableApnsToken',
     'disableGcmToken',
+    'disableAllGcmTokens',
```

Comment:
In any case, we made some fixes so that the state is cleared b/w tests (https://github.com/segment-integrations/analytics.js-integration-outbound/pull/7). Could you rebase your changes on top of that and see if the issue is fixed?